### ISSUE TITLE
[Feature Request] Scheduling Matches

### ISSUE THREAD

**Issue by xe1os** [PhlexPlexico/get5-web#92]
Hi @PhlexPlexico,

Having a feature of auto starting a match by scheduling it on a certain date and time via web panel.

**Comment by PhlexPlexico** [comment]
That would require a bit of rework for what is currently implemented. Please use the provided template for the feature request by the way, it's easier for me to dissect what's needed :)

As for this request - I think this would be done long down the road, as I'd need to change out matches are setup/stored in the database (server ID cannot be null, for example). At that point, I'd want to rewrite the app from scratch if I had to go back and do that, haha. But I'll keep this here and backlog it so I can reference it later!

**Comment by TandelK** [comment]
This will require complete rework most probably on cronjobs with Docker system.

**Comment by PhlexPlexico** [comment]
I don't think you would need anything external, as that would add more reliance on what a user needs. I would prefer if it was contained within python itself. I probably won't be working on this, but I would imagine you could use an event scheduler somehow. But I'm still working more towards tournament use and leagues where admins are currently present with this for now. Not looking towards large scale leagues just yet.

**Comment by PhlexPlexico** [comment]
Looking through some forks since there's been an uptick of activity on these projects, it seems someone made a wait-timer for cancelling matches if no one joins them and updates the database. Seems like this could be doable by adding a schedule match if someone were to take up a task of doing scheduled matches: https://github.com/nishuo/get5-web/commit/3fa0f3aabee82b05108e335383cc418ac8c69612

**Comment by TandelK** [comment]
yeah i have tested that code before and it was working on splewis verison as i had worked with nishuo on his first deployment .
